gae-interop-testing/README.md

```diff
@@ -0,0 +1,50 @@
+Google App Engine interop tests
```

Comment:
This dir should not be exported when cutting a release I think.

Reply:
Does anything special need to be done to avoid including this dir?

Reply:
Yes, see https://github.com/grpc/grpc-java/commit/4988d8be6b47eb65b3d2c3628dba50c37c991ce6